src/Components/NavBar.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import styled from 'styled-components';
+
+
+
+const NavBar = () => {
+	return (
+		<Nav>
+			<img>
+			</img>
```

Comment:
Once we have a logo we can add that to the nav bar here. Will need to make sure to make it a self closing tag.